docker/ml-testing-toolkit/spec_files/system_config.json

```diff
@@ -9,31 +11,52 @@
     "userSettings": 1
   },
   "DB": {
-    "URI": "mongodb://mongo:27018/dfsps"
+    "URI": "mongodb://ttk:ttk@localhost:27017/ttk"
   },
   "OAUTH": {
     "AUTH_ENABLED": false,
-    "APP_OAUTH_CLIENT_KEY": "asdf",
-    "APP_OAUTH_CLIENT_SECRET": "asdf",
+    "APP_OAUTH_CLIENT_KEY": "ttk",
+    "APP_OAUTH_CLIENT_SECRET": "secret",
     "MTA_ROLE": "Application/MTA",
     "PTA_ROLE": "Application/PTA",
     "EVERYONE_ROLE": "Internal/everyone",
-    "OAUTH2_ISSUER": "http://ml-testing-toolkit:5050/api/oauth2/token",
-    "JWT_COOKIE_NAME": "TTK-API_ACCESS_TOKEN",
-    "EMBEDDED_CERTIFICATE": "password"
+    "OAUTH2_TOKEN_ISS": "http://mojaloop-testing-toolkit:5050/api/oauth2/token",
+    "OAUTH2_ISSUER": "http://mojaloop-testing-toolkit:5050/api/oauth2/token",
+    "EMBEDDED_CERTIFICATE": "CERTIFICATE"
   },
   "CONNECTION_MANAGER": {
+    "ENABLED": false,
     "API_URL": "http://connection-manager-api:5061",
     "AUTH_ENABLED": false,
     "HUB_USERNAME": "hub",
     "HUB_PASSWORD": "hub"
   },
   "KEYCLOAK": {
     "ENABLED": false,
-    "API_URL": "http://172.17.0.1:9080",
+    "API_URL": "http://localhost:8080",
```

Comment:
I dont think this will work on mac.

Can you not use the service name instead?

Or are you trying to hit a service locally?

Reply:
HI Miguel, these are just dummy values.